### DISCUSSION TITLE
Does windows app sdk only work with C++?

### DISCUSSION THREAD

**Discussion by BraveEvidence** [microsoft/WindowsAppSDK#2535] [Q&A] [answered]
I am newbie in windows app development.

If I am using winui and C# is it possible to integrate windows app sdk and use it in C# completely?

**Comment by castorix** [comment, marked as answer]
Windows App SDK is used in C++ and C# : [Downloads for the Windows App SDK](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads)
(it can also be used in VB.NET, but with some limitations)

**Reply by BraveEvidence** [reply]
Thanks
